pandas/io/sql.py

```diff
@@ -404,7 +404,7 @@ def to_sql(frame, name, con, schema=None, if_exists='fail', index=True,
 
     Parameters
     ----------
-    frame : DataFrame
+    frame : DataFrame, Series
```

Comment:
@jorisvandenbossche is this right? should we deprecate / rename this parameter?

Reply:
yes, this seems right, as a few lines below it does:

```
    if isinstance(frame, Series):
        frame = frame.to_frame()
```

but since this is mostly an internal method (most people will directly use DataFrame/Series.to_sql), I don't think we should bother with deprecating or renaming it